lib/helper/Playwright.js

```diff
@@ -2705,7 +2709,16 @@ class Playwright extends Helper {
   /**
    * Blocks traffic for URL.
    *
-   * @param url URL to block . URL can contain * for wildcards. Example: https://www.example.com* to block all traffic for that domain.
+   * Examples:
+   *
+   * ```js
+   * await I.blockTraffic('http://example.com/css/style.css');
```

Comment:
do we always need await?

Reply:
No. We don't need actually.
Good catch.
I made hands-on tests and they all worked just fine without `await`.

-> I updated the examples in docblock. I didn't yet change it in the functional code to make the methods sync, that don't need to be async. A
ctually almost all new steps are sync methods except `await I.seeTraffic`
`seeTraffic` is async because we decided to have waiting mechanism in there for up to 10 seconds (default value).